File: library/src/main/java/com/example/library/dto/response/CategorySyncResponse.java
package com.example.library.dto.response;


import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CategorySyncResponse {

    private Integer id;

    private String categoryCode;

    private String categoryName;

    private Integer createdBy;
        
    private Long version;

    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt;
    
}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/CategorySyncResponse.kt
package com.example.myapplication.data.remote.dto.response

import java.time.LocalDateTime

data class CategorySyncResponse(
    val id: Int,
    val categoryCode: String,
    val categoryName: String,
    val createdBy: Int,
    val version : Long,
    val updatedAt: LocalDateTime,
    val deleteAt: LocalDateTime?
)